docs/wiki/Known issues.md

```diff
@@ -45,9 +45,17 @@ Further details are also provided in issue [#1342](https://github.com/Azure/Reso
 
 ## Microsoft.RecoveryServices/vaults
 
-The Recovery Services Vaults module does not currently validate the identity property (system or user assigned identity).
+The Recovery Services Vaults module does not currently attach the content of the identity property correctly when both user- and systemassigned identity fields are selected.
 
-The module pipeline fails in the deployment validation step when system and user assigned identity parameters are added as input parameters.
+The pipeline shows a success but the assignment of both identities never happens although both identities (systemassigned is a serviceprincipal, userassigned is a managed identity resource) get created successfully.
+
+Upon cleanup the system-assigned identity will not be removed.
+
+When the deployment is then run again it fails, because Azure tries to attach this rogue service principal as a system-assigned identity.
+
+The branch `users/fblix_jpeasier/rsvIdentity` has been used to testing and contains failing and succeeding samples.
```

Comment:
I'd suggest to remove this paragraph. The referenced dev branch will likely get stale going forward.

Reply:
agreed!